src/hooks/useMutation.test.ts

```diff
@@ -0,0 +1,7 @@
+describe("useMutation()", () => {
```

Comment:
@kei95 Sorry I was stack with writing test. I am going to work soon in another PR

Reply:
Np! I think you can even remove this hook and test and make the tweak on `useFetch()` is sufficient.